modules/ui/entity_editor.js

```diff
@@ -142,10 +142,10 @@ export function uiEntityEditor(context) {
             if (priorActivePreset && _activePresets.length === 1 && priorActivePreset !== _activePresets[0]) {
                 // flash the button to indicate the preset changed
                 context.container().selectAll('.entity-editor button.preset-reset .label')
-                    .style('background-color', '#fff')
-                    .transition()
-                    .duration(750)
-                    .style('background-color', null);
+                    .classed('flash-bg', true)
+                    .on('animationend', function() {
+                        this.classList.remove('flash-bg');
+                    });
```

Comment:
maybe using d3 wrapper functions could make this slightly more readable (as it mirrors the `.classed('flash-bg', true)` on the previous line)?

```suggestion
                    .on('animationend', function() {
                        d3_select(this).classed('flash-bg', false));
                    });
```

Reply:
done.